server/server.go

```diff
@@ -41,13 +46,25 @@ type Config struct {
 	// Mux is the http.ServeMux on which the server registers its HTTP
 	// handlers.
 	Mux *http.ServeMux
+	// BackupBucket is an AWS S33 bucket name to which database
```

Comment:
S3

Reply:
I'm sorry, my temporal synchromesh got out of skew and I wrote this comment in the far future. Yeah, it's just regular S3 for now.

Reply:
Readjusted my oscillation overthruster, temporal incidents should not reoccur.

Reply:
Where are we going? 

Planet Ten!

When?

Real soon!